README.md

```diff
@@ -76,14 +76,36 @@ go build arduino.cc/arduino-builder
 In order to run the tests, type:
 
 ```
-go test -timeout 60m -v ./src/arduino.cc/builder/test/...
+go test arduino.cc/builder/test
+```
+
+This runs all tests, showing any failures and a summary at the end.
+Add the -v option to show each test as it is being ran. Currently,
+arduino-builder itself also generates copious output, even for
+non-failing testcases and without -v, and testing does not stop at the
+first failure, so you probably want to redirect test output so you can
+scroll back to find any failures.
+
+To run a single test, use the -run option, which accepts a regular
+expression (see also go help testflag).
+
+```
+go test arduino.cc/builder/test -run 'TestBuilderEmptySketch'
+go test arduino.cc/builder/test -run 'TestPrototypesAdder.*'
 ```
 
 In jenkins, use
 ```
-go test -timeout 60m -v ./src/arduino.cc/builder/test/... | bin/go-junit-report > report.xml
+go test -v arduino.cc/builder/test | bin/go-junit-report > report.xml
```

Comment:
Well this timeout has been added not much for the initial download of tools but mainly for the ARM release: running the test natively on a Raspberry PI2 takes about 30 minutes. So probably it's best to keep it as default?